### PR TITLE
feat(movie-web): add working instances and watching status

### PR DESCRIPTION
**Pull request by myysty**
## Description 

sudo-flix is gone, added working instances, added watching status and use show/movie name for presence name.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image
presence name update:
image

</details>

### CODE REVIEW

**Review by DarkVillager — APPROVED** [review]
Run prettier other than that LGTM

**Comment by DarkVillager** [comment]
@myysty attempt the following cmd: npm run format
After make sure to discard non movie-web changed files

**Comment by DarkVillager** [comment]
Interesting. Have you run: npm i
before?

**Comment by myysty** [comment]
yes i have

File: websites/M/movie-web/presence.ts [hunk 1]
```diff
@@ -107,7 +109,7 @@ presence.on("UpdateData", async () => {
 			presenceData.smallImageText = "Loading";
 		} else if (controls.isPlaying) {
 			[, presenceData.endTimestamp] = presence.getTimestampsfromMedia(
-				document.querySelector("video")
+				document.querySelector("video"),
```
Comment (DarkVillager) [inline websites/M/movie-web/presence.ts:112]:
```suggestion
				document.querySelector("video")
```

File: websites/M/movie-web/presence.ts [hunk 2]
```diff
@@ -142,7 +144,7 @@ function createProgressBar(
 		numChars = Math.floor((progress / 100) * barLength);
 
 	return `${barFill.repeat(numChars)}${barTrack.repeat(
-		barLength - numChars
+		barLength - numChars,
```
Comment (DarkVillager) [inline websites/M/movie-web/presence.ts:147]:
```suggestion
		barLength - numChars
```

File: websites/M/movie-web/presence.ts [hunk 3]
```diff
@@ -132,7 +134,7 @@ function createProgressBar(
 		barTrack: string;
 		barFill: string;
 		showLabel: boolean;
-	}
+	},
```
Comment (DarkVillager) [inline websites/M/movie-web/presence.ts:137]:
```suggestion
	}
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]
This _should_ fix it

**Review by theusaf — APPROVED** [review]
LGTM

**Review by Bas950 — APPROVED** [review]

**Pull request merged**